pandas/tests/io/pytables/test_store.py

```diff
@@ -91,7 +90,7 @@ def test_context(self, setup_path):
         except ValueError:
             pass
         finally:
-            safe_remove(path)
+            tm.ensure_clean(path)
```

Comment:
wouldnt we want to use tm.ensure_clean as a context?

Reply:
I'm a little stuck actually on how to progress here. I'd be keen on putting in the extra time to get this PR cleaned up. If you could take a look & review my most recent commit that would be greatly appreciated. Thanks for the help.

Reply:
to use tm.ensure_clean as a context, it would look like:

```
with tm.ensure_clean(path):
    do_stuff
```

as opposed to

```
try:
    do_stuff
finally:
    tm.ensure_clean(path)
```